app/components/SheetMusic/comments/Comment.jsx

```diff
@@ -0,0 +1,52 @@
+
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+
+let Comment = React.createClass({
```

Comment:
I recommend using `export default React.createClass` to be consistent with other files.